ui/app/AppLayouts/Chat/controls/community/InDropdown.qml

```diff
@@ -154,12 +148,12 @@ StatusDropdown {
 
             visible: root.allowChoosingEntireCommunity
         }
-
         StatusScrollView {
             id: scrollView
-
             Layout.fillWidth: true
-            Layout.fillHeight: true
+            Layout.minimumHeight: (((topRepeater.count > 5) ? 5 : topRepeater.count)*44)
```

Comment:
Since 44 is the same value than list item `Layout.preferredHeight` could we use a `d` property instead and share accross the needed properites?

Reply:
Same with the 5 value. Could we use it as `d` property if it is a `default` margin?